test/common/performance/aggregator/aggregator.go

```diff
@@ -184,7 +218,7 @@ func (ag *Aggregator) Run(ctx context.Context) {
 		if ag.publishResults {
 			sendLatency := timestampAccepted.Sub(timestampSent)
 			// Uncomment to get CSV directly from this container log
-			//fmt.Printf("%f,%d,\n", mako.XTime(timestampSent), sendLatency.Nanoseconds())
+			// fmt.Printf("%f,%d,\n", mako.XTime(timestampSent), sendLatency.Nanoseconds())
```

Comment:
Would it make sense to have a TODO (or an issue) to track how to get these metrics without recompiling? Something like a flag, or a context or using a different logger. Not related to this PR, just made me think of it.

Reply:
Good point. Added the TODO comment.
Thanks!